src/s3.js

```diff
@@ -0,0 +1,7 @@
+function createS3URL(bucket, key) {
+  return new URL(`s3://${bucket}/${key}`)
```

Comment:
Shouldn't this include region?

Reply:
So... I'm not sure. But I think it is ok not to? 

Just dug around and learned
* The awsRegion is probably available (as `event.Records[].awsRegion) in the lambda when I call this fn ([docs](https://docs.aws.amazon.com/lambda/latest/dg/with-s3.html))
* Quite a few aws docs omit region from s3 bucket identifiers
  * "Accessing a bucket using s3 format" - https://docs.aws.amazon.com/AmazonS3/latest/userguide/access-bucket-intro.html#accessing-a-bucket-using-S3-format
     > Some AWS services require specifying an Amazon S3 bucket using S3://bucket. The following example shows the correct format. Be aware that when using this format, the bucket name does not include the AWS Region.
     >
     > S3://bucket-name/key-name
  * s3 arns - https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-arn-format.html
     > An Amazon S3 ARN excludes the AWS Region and namespace, but includes the following:
     >
     >    Partition ‐ aws is a common partition name. If your resources are in the China (Beijing) Region, aws-cn is the partition name.
     >
     >    Service ‐ s3.
     >
     >    Relative ID ‐ bucket-name or a bucket-name/object-key. You can use wild cards.
     >
     > The ARN format for Amazon S3 resources reduces to the following:
     >
     > arn:aws:s3:::bucket_name/key_name

I'm happy to do whatever. Notably aws doesn't really document an identifier that *does* include the region, so I'm inclined to 'drop it'. otoh I'm sympathetic to the argument that using the `arn:aws:s3` URIs seems like it might be more preferred/standard on their end (instead of the `s3:` uris). happy to do whatever